src/pages/[platform]/start/manual-installation/index.mdx

```diff
@@ -28,7 +28,34 @@ export function getStaticProps(context) {
   };
 }
 
-The easiest way to get started with AWS Amplify is through [npm](https://npmjs.com) with [`create-amplify`](https://www.npmjs.com/package/create-amplify), which is the workflow our [quickstart](/[platform]/start/quickstart) recommends. However, for some use cases, it may be preferable to start from scratch, either with a brand new directory or an existing frontend app. This guide will walk you through how to initialize your project, install dependencies, and author your first backend.
+To get started with AWS Amplify we recommend that you use our our [quickstart](/[platform]/start/quickstart) starter template. However, for some use cases, it may be preferable to start from scratch, either with a brand new directory or an existing frontend app. In that case we recommend to use [npm](https://npmjs.com) with [`create-amplify`](https://www.npmjs.com/package/create-amplify). 
+
+```bash showLineNumbers={false}
+npm create amplify@latest
+```
+
+```console showLineNumbers={false}
+? Where should we create your project? (.) # press enter
+```
+
+Running this command will scaffold a lightweight Amplify project in your current project with the following files:
+
+```text
+├── amplify/
+│   ├── auth/
+│   │   └── resource.ts
+│   ├── data/
+│   │   └── resource.ts
+│   ├── backend.ts
```

Comment:
```suggestion
│   ├── backend.ts
│   ├── tsconfig.json
```

Reply:
sneaking this in 😉

Reply:
👍

Reply:
[6bd7c20](https://github.com/aws-amplify/docs/pull/7463/commits/6bd7c20477c0232aa44fae1b639ea19e90ca8e4f)